app/src/main/java/io/gnosis/safe/ui/transactions/execution/TxReviewViewModel.kt

```diff
@@ -162,14 +179,14 @@ class TxReviewViewModel
                     if (!userEditedFeeData) {
                         nonce = minNonce
                         gasLimit = estimationParams.estimate
-                        maxPriorityFeePerGas = Wei(BigInteger.valueOf(DEFAULT_MINER_TIP)).toGWei(activeSafe.chain.currency.decimals)
-                        maxFeePerGas = Wei(gasPrice).toGWei(activeSafe.chain.currency.decimals).plus(maxPriorityFeePerGas!!)
+                        maxPriorityFeePerGas =
+                            Wei(BigInteger.valueOf(DEFAULT_MINER_TIP)).toGWei(activeSafe.chain.currency.decimals)
+                        maxFeePerGas = Wei(gasPrice).toGWei(activeSafe.chain.currency.decimals)
```

Comment:
gasPrice is equivalent to the maxFeePerGas. I think this would charge more gas than needed by the estimation. Can you double-check please? https://ethereum.org/en/developers/docs/gas/#what-was-the-london-upgrade-eip-1559

Reply:
Here gasPrice is equivalent to base fee and maxFeePerGas is priority fee added to base fee as specified in eip-1559. The calculation should be correct. I will add some comments to make it more clear.